Source/ablastr/parallelization/MPIInitHelpers.H

```diff
@@ -15,7 +15,7 @@ namespace ablastr::parallelization
      *
      * @return the MPI_THREAD_* level required for MPI_Init_thread
      */
-    int
+    constexpr int
```

Comment:
Sure, possible given the implementation.

In practice, never compile-time because the `.cpp` file is not seen in other TUs.

Reply:
`mpi_thread_required()` is called only twice in the code, inside `MPIInitHelpers.cpp` .
With my latest commit I've changed 
` int thread_required = mpi_thread_required();` into `constexpr int thread_required = mpi_thread_required();`, to make sure that the function is computed at compile-time.

Reply:
Ah, inside `MPIInitHelpers.cpp` the implementation is know. Makes sense.